java/src/org/openqa/selenium/devtools/DevTools.java

```diff
@@ -73,6 +73,7 @@ cdpSession, getDomains().target().detachFromTarget(Optional.of(id), Optional.emp
         // Exceptions should not prevent closing the connection and the web driver
         log.warning("Exception while detaching from target: " + e.getMessage());
       }
+      cdpSession = null;
```

Comment:
This is the only bit I do not understand. Why is this needed?

Reply:
I did that while testing locally for a failing test. Turns out, it is not needed. Thank you for pointing it out.